pages/main-experience.tsx

```diff
@@ -0,0 +1,54 @@
+import Button from "components/button/button"
+import CustomDivComponent from "components/CustomDivComponent"
+
+function MainExperience() {
+  return ( <>
+    <CustomDivComponent 
+    className = "sampleDiv"
+    color = "green"
+    header = "first div"
+    height = "500px"
+    width = "100%"
+    >
+      blah
+      <Button onClick={() => console.log("you clicked the cat!")}
+        backgroundURL = "/cat.png"
+        height = "200px"
+        width = "200px"
+        > 
+      </Button>
+    </CustomDivComponent>
+    <CustomDivComponent 
+    className = "sampleDiv2"
+    color = "pink"
+    header = "second div"
+    height = "500px"
+    width = "100%"
+    >
```

Comment:
let's also indent the props here and anywhere else this pattern is found in this PR